priv/repo/migrations/20201012053216_create_task.exs

```diff
@@ -0,0 +1,10 @@
+defmodule Todolist.Repo.Migrations.CreateTask do
```

Comment:
It's good to be as descriptive as possible. For now leave it as it is but something like "CreateTasksTable" says a lot more.

Reply:
Done